jsapp/js/components/account/accountSidebar.tsx

```diff
@@ -83,7 +83,6 @@ export default class AccountSidebar extends React.Component<
           {window.MFAEnabled && <bem.FormSidebar__label
             m={{selected: this.isSecuritySelected()}}
             href={'#' + ROUTES.SECURITY}
-            disabled
           >
             {/*TODO: get the data usage icon*/}
```

Comment:
Are all those TODO comments still valid? Do you have these icons from Ignacio? Do you want me to add them?